wiki/content/slash-graphql/slash-quick-start.md

```diff
@@ -2,16 +2,30 @@
 title = "Slash Quick Start"
 [menu.main]
     parent = "slash-graphql"
-    weight = 1   
+    weight = 1
 +++
 
-*These are draft docs for Slash GraphQL, which is currently in beta*
+## Introduction
 
-Welcome to Slash GraphQL. By now, you should have created your first deployment, and are looking for a schema to test out. Don't worry, we've got you covered.
+Welcome to [Slash GraphQL](https://dgraph.io/slash-graphql), the world’s most advanced, hosted GraphQL backend. By now, you should have created your first deployment, and you're looking for a quick-start guide to test it out. Don't worry, in this document we got you covered.
 
-This example is for todo app that can support multiple users. We just have two types: Tasks and Users.
+In this guide, we will create a database for a small app from the ground up using Slash GraphQL. The easy steps you will learn in this tutorial are fundamental to use Slash GraphQL, and mastering it will give you a better understanding of the powerful features available.
```

Comment:
Please swap:

"The easy steps you will learn in this tutorial are fundamental to use Slash GraphQL, and mastering it will give you a better understanding of the powerful features available."

With:

"The easy steps you will learn in this tutorial are fundamental to using Slash GraphQL, and mastering them will give you a better understanding of the powerful features available." (use -> using; it -> them)

Reply:
oops, I'll fix it! 👍